Apex Language/ClassStudentMapService.apxc

```diff
@@ -0,0 +1,14 @@
+public class ClassStudentMapService {
+    public static Map<Id,Set<Id>> getMapOfClassStudentSet(){
+        Map<Id,Set<Id>> MapOfClassStudentSetId = new Map<Id,Set<Id>>();
```

Comment:
You can directly return Map of id and class object instead of iterating them separately

Reply:
Done suggested changes.